pkg/ovs/openflow/ofctrl_meter.go

```diff
@@ -0,0 +1,133 @@
+// Copyright 2020 Antrea Authors
```

Comment:
```suggestion
// Copyright 2021 Antrea Authors
```
It should the year the file is first published

Reply:
Done.